cloud_Azure/terraform/module/README.md

```diff
@@ -53,24 +59,18 @@ To install python and its requirements:
 * [Install pip3](https://pip.pypa.io/en/stable/installing/)
 * Install packages: run `pip3 install -r ../../requirements.txt` in example directory
 
-## Providers
-
-| Name | Version |
-|------|---------|
-| azurem | >= =2.20.0 |
-| null | >= 2.1.2 |
-| external | >= 2.0.0 |
-
 ## Inputs
 
 | Name | Description | Type | Default | Required |
 |------|-------------|------|---------|:--------:|
 | location | Azure location of the resources to gather logs | `string` | `` | yes |
 | subscription_id | Id of the subscription in which resource are located | `string` | `` | yes |
 | resource_group_name | Name of the resource group to gather logs from | `string` | `` | yes |
-| principal_id | Id of the Service Principal Id for kentik app connection | `string` | `` | yes |
-| prefix| Prefix for the naming resources created by this module | `string` | `kentik` | no |
-
+| prefix| Prefix for the naming resources created by this module | `string` | `` | yes |
```

Comment:
This prefix should be unique as it is used to create StorageAccount.name, which must be Azure-wide unique.
Do you want to use UUID generator to generate the name?

Reply:
I wonder how users are going to select the prefix. We definitely cannot use an ID generated dynamically on each invocation (because that would interfere with Terraform state management). On the other hand, users may run into conflicts when using manually generated prefixes, especially in large organizations. Could a synthetic,  name (for example) "kentik.subscription_id.resource_group_name" work (at least as a default, which users can override)? I'd guess that resource groups names must be unique within a subscription and subscriptions must be unique Azure-wide.

Reply:
Will improve StorageAccount.name solution in next PR (multiple subscriptions support)